examples/dotnet/SeleniumDocs/SeleniumManager/UsageTest.cs

```diff
@@ -0,0 +1,20 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium.Chrome;
+
+namespace SeleniumDocs.SeleniumManager
```

Comment:
`SeleniumManager` is clashing with the type of the same name in selenium 

```suggestion
namespace SeleniumDocs.SeleniumManagerTest
```

Reply:
(the name I put is just a suggestion)

Reply:
It is not a problem.